agenda-webapps/src/main/webapp/vue-app/agenda-common/components/event/view/AgendaEventIcs.vue

```diff
@@ -50,15 +57,37 @@ export default {
         return result + line;
       };
 
+      const foldDescription = (text) => {
```

Comment:
Could you check this [code ](https://github.com/exoplatform/task/blob/23d60150f19b08dec8f519c61b9cf856f27fefa3/task-management/src/main/webapp/vue-app/tasks-management/components/Project/ProjectCardFront.vue#L287) on how to convert HTML to plain text ?